readme.md

```diff
@@ -174,10 +190,30 @@ console.log(chalk.styles.red.open + 'Hello' + chalk.styles.red.close);
 - `bgWhite`
 
 
-## 256-colors
+## 256/16 million (Truecolor) color support
+
+Chalk supports 256 colors and, when manually specified, [Truecolor (16 million colors)](https://gist.github.com/XVilka/8346728) on all supported terminal emulators.
```

Comment:
Why not? It's a qualifier for the subsequent clause. Grammatically the lack of a comma wouldn't make sense here.